using_images/other_images/jenkins.adoc

```diff
@@ -52,18 +52,96 @@ You can find https://github.com/openshift/origin/tree/master/examples/image-stre
 
 === Initializing Jenkins
 
+There are two methods for managing authentication with the {product-title} Jenkins images when they are initially started.
+
+* Standard authentication provided by Jenkins
+
+* {product-title} OAuth authentication provided by the OpenShift Login plugin.
+
+==== Jenkins standard authentication
+
+Jenkins authentication will be used by default with the {product-title} Jenkins images if the image is run outside of {product-title}.
+
 The first time you start Jenkins, the configuration is created along with the administrator user and password.
-The default login is `*admin/password*`.
+The default user login is `*admin/password*`.
 The default password can be configured by setting the `*JENKINS_PASSWORD*` environment variable.
 
 The following command creates a new Jenkins xref:../../architecture/core_concepts/pods_and_services.adoc#pods[pod] with Jenkins running in a container:
 
+If you choose to run with standard Jenkins authentication while the {product-title} Jenkins image runs inside of {product-title}, the the password for
+`*admin*` will be a randomly generated string.  If you manually instantiate a template to launch Jenkins, the password will be noted in the output.
```

Comment:
it won't be a randomly generated string unless they instantiate it from a template.  if they just run the image w/o setting JENKINS_PASSWORD (which is what the template does), it'll still be "password", right?